swagger.yaml

```diff
@@ -79,6 +73,128 @@ paths:
       security:
       - oauth2:
         - API
+  '/ereignisse/{vorgangsNummer}':
+    get:
+      tags:
+        - Vorgangsereignisse
+      summary: alle Ereignisse eines Vorgangs abrufen
+      description: >-
+        Als Parameter wird die Vorgangsnummer verwendet, die aus BaufiSmart
+        erzeugt wurde.
+      operationId: getEreignisse
+      produces:
+        - application/json;charset=UTF-8
+      parameters:
+        - name: vorgangsNummer
+          in: path
+          description: Nummer des Vorgangs
+          type: string
+          required: true
+      responses:
+        '200':
+          description: OK
+          schema:
+            type: array
+            items:
+              $ref: '#/definitions/Ereignis'
+        '401':
+          description: Unauthorized
+          schema:
+            $ref: '#/definitions/Error'
+        '403':
+          description: Forbidden
+          schema:
+            $ref: '#/definitions/Error'
+        '404':
+          description: Not Found
+          schema:
+            $ref: '#/definitions/Error'
+        '405':
+          description: Method Not Allowed
+          schema:
+            $ref: '#/definitions/Error'
+        '500':
+          description: Internal Server Error
+          schema:
+            $ref: '#/definitions/Error'
+        '502':
+          description: Bad Gateway
+          schema:
+            $ref: '#/definitions/Error'
+        '504':
+          description: Gateway Timeout
+          schema:
+            $ref: '#/definitions/Error'
+      security:
+        - oauth2:
+            - API
+
+  '/ereignisse/{vorgangsNummer}/{antragsNummer}/{teilAntragsNummer}':
```

Comment:
Bei der Terminologie der Parameter wird es in naher Zukunft nochmal Änderungen geben.
Dies muss aber noch entschiede werden, sodass es dann umgesetzt werden kann.